File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height, row, cols, space ;
    do
    {
        height = get_int ("Height: ");
    }
    while (height < 1 || height > 8) ;

    for (row = 0; row < height; row++)
    {
        for (space = 0; space < height - row - 1; space++)
        {
            printf(" ");
        }
        for (cols = 0; cols <= row; cols++)
        {
            printf ("#");
        }
        printf ("  ");
        for (cols = 0; cols <= row; cols++)
        {
            printf ("#");
        }
        printf ("\n");
    }

}

File: sentimental-mario-more/mario.py
def main():
    while True:
        height_str = input("Height: ")
        if height_str.isdigit():
            height = int(height_str)
            if height >= 1 and height <= 8:
                break
        print("Invalid input. Please enter a numeric value between 1 and 8.")

    for row in range(height):
        for space in range(height - row - 1):
            print(" ", end="")
        for cols in range(row + 1):
            print("#", end="")
        print("  ", end="")
        for cols in range(row + 1):
            print("#", end="")
        print()

if __name__ == "__main__":
    main()

